integrations/jira/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,35 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: project
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .key | tostring
+          title: .name | tostring
+          blueprint: '"project"'
+          properties:
+            url: .self
+  - kind: issue
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .key | tostring
+          title: .fields.summary
+          blueprint: '"issue"'
+          properties:
+            url: .self
```

Comment:
Is it api url or web url?

Reply:
API URL, their API does not return the web URL

Reply:
Can we maybe construct it ourselves?